code/week2/week2.cpp

```diff
@@ -0,0 +1,85 @@
+// make sure to compile with g++
+// to compile do
+//      g++ -o week2.out week2.cpp operations.cpp
```

Comment:
We can probably make this 
`g++ -o week2.out *.cpp`
so that the command is shorter and we don't need to change it all the time.